### ISSUE TITLE
export TextDocumentItem

### ISSUE THREAD

**Issue by siegebell** [Microsoft/vscode-languageserver-node#115]
main.d.ts should export `TextDocumentItem`

**Comment by felixfbecker** [comment]
Need this too

**Comment by Turbo87** [comment]
@dbaeumer this seems to have been fixed with the v3.0.3 release

**Comment by dbaeumer** [comment]
@Turbo87 Thanks. This is correct. The client now has:
```ts
export * from 'vscode-languageserver-types';
export * from './protocol';
```
